### PR TITLE
Enable importing of container name-only images and user-rules via sdscli

### PR DESCRIPTION
**Pull request by joshgarde**
Ticket: https://hysds-core.atlassian.net/browse/HC-517

Add support in sdscli for container specifications which do not specify a container_url and only specify a container_name. This behavior is already supported within hysds-core and only requires a tooling change to the sds pkg import command.

https://github.com/hysds/hysds/blob/944d3a89997b1aff416dc627b095a6b5e10b2242/hysds/containers/base.py#L279-L283 

Additionally, add support for importing user rules from a package if part of the package.

### CODE REVIEW

**Comment by mcayanan** [comment]
This PR looks straight forward enough but would like a 2nd pair of eyes on this to make sure we're not impacting anything with what core is doing now. I couldn't see any issues. Looks like the changes allow more flexibility to not expect certain fields when importing a package.

**Comment by DustinKLo** [comment]
LGTM, but lets get another pair of eyes

**Comment by mcayanan** [comment]
@joshgarde Could you please bump up the version in https://github.com/sdskit/sdscli/blob/59874cf04efa6ac016629925c12a0196a612d2a2/sdscli/__init__.py#L6 to `1.3.2`? This will allow us to notate any specific updates when we cut the next HySDS Core Release. Thanks!

**Review by pymonger — COMMENTED** [review]
LGTM pending the version bump

**Comment by joshgarde** [comment]
Version bumped

**Review by mcayanan — APPROVED** [review]

**Pull request merged**
